src/fairchem/core/trainers/ocp_trainer.py

```diff
@@ -79,9 +79,7 @@ def __init__(
         loss_functions: dict[str, str | float],
         evaluation_metrics: dict[str, str],
         identifier: str,
-        # TODO: dealing with local rank is dangerous
-        # T201111838 remove this and use CUDA_VISIBILE_DEVICES instead so trainers don't need to know about which devie to use
-        local_rank: int,
+        local_rank: int,  # DEPRECATED, DO NOT USE
```

Comment:
can we just remove this altogether and let if fail if used? or does it cause deeper issues if we do?